build/Targets/Versions.props

```diff
@@ -49,6 +49,8 @@
     <NuGetPackageRoot Condition="'$(NuGetPackageRoot)' == '' AND '$(OS)' == 'Windows_NT'">$(UserProfile)/.nuget/packages/</NuGetPackageRoot>
     <NuGetPackageRoot Condition="'$(NuGetPackageRoot)' == '' AND '$(OS)' != 'Windows_NT'">$(HOME)/.nuget/packages/</NuGetPackageRoot>
     <NuGetPackagesDirectory>$(NuGetPackageRoot)</NuGetPackagesDirectory>
+
+    <Version Condition="$(Version) == ''">$(NuGetPerBuildPreReleaseVersion)</Version>
```

Comment:
This is a dangerous property to set.  NuGet uses it for a lot of different scenarios.  Why do we need it here?